packages/dm_base/blueprints/wiki/routes.py

```diff
@@ -19,11 +19,11 @@ def index():
 @blueprint.route('')
 def index_sub(sub):
     return render_template('index_docsify.html')
-
+    
```

Comment:
please clean whitespace. I suggest you use a linter to show you these errors, will make your life a lot easier